### DISCUSSION TITLE
the bearing arrow cursor is confusing

### DISCUSSION THREAD

**Discussion by alexbodn** [osmandapp/OsmAnd#13474] [General]
hello friends,

the device direction is being showed by a small blue lantern beam. 
I'm usually watching the map in fixed north, oriented manually by this lantern beam.
there's i problem i don't seem to solve on my own: I'd like to better see this lantern beam.
but this lantern beam is so small, that it is  almost entirely hidden under the bearing arrow.
that wouldn't be a problem when moving, but the arrow is shown flickering while standing, when the direction cursor becomes the only needed.
the solution could be a way to switch the bearing arrow off altogether, a way to make the lantern beam more prominent, like in google maps, or contrast better with the bearing arrow.

thanks in advance for any solution,
alex

**Comment by barteis** [comment]
For walking , say:

Try Settings / Walking / Profile appearance,  then scroll down to "Position icon at rest". There are three beam sizes to choose from.
